tests/perf-test/SelectionList.perf-test.tsx

```diff
@@ -99,20 +107,22 @@ test('[SelectionList] should render 1 section and a thousand items', () => {
 });
 
 test('[SelectionList] should press a list item', () => {
-    const scenario = (screen) => {
+    const scenario = (screen: RenderResult) => {
         fireEvent.press(screen.getByText('Item 5'));
     };
 
+    // @ts-expect-error scenario is expected to be an async function
     measurePerformance(<SelectionListWrapper />, {scenario});
```

Comment:
Another way to handle it is
```suggestion
    // eslint-disable-next-line @typescript-eslint/require-await
    const scenario = async (screen: RenderResult) => {
        fireEvent.press(screen.getByText('Item 5'));
    };
    
    measurePerformance(<SelectionListWrapper />, {scenario});
```